### PR TITLE
Changed the image and replaced it with text in "post link to your repository in chat"

### PR DESCRIPTION
**Pull request by lorugant**
Changes made according to #1343 and did not change the last screenshot as a PR is already in progress #1331 
Also included the profile link which I found recently in newer PRs. Please let me know if any other changes are needed. Thanks!!

[rawgit](https://rawgit.com/lorugant/lorugant.github.io/lBranch/index.html#!./pages/vi/vi-github-and-markdown.md)

### CODE REVIEW

File: pages/vi/vi-github-and-markdown.md [hunk 1]
```diff
@@ -66,7 +66,9 @@ After you fork the repository, you will be on your page: &lt;username&gt;/open-l
 
 After you rename it, make sure to post a link to the github.io website you've just created (&lt;username&gt;.github.io) in the [gitter.im chat](https://gitter.im/open-learning-exchange/chat).
 
-![Example of github.io page](images/vi-example-githubio.png)
+> ** @dogi  please look at
+https://YourUserName.github.io and
+ https://rawgit.com/YourUserName/YourUserName.github.io/YourBranchName/#!pages/vi/profiles/YourUserName.md **
```
Comment (xyb994) [inline pages/vi/vi-github-and-markdown.md]: As we [discussed in the other PR](https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/1343#discussion_r147562721), the personal profile markdown file wouldn't exist at this point if we follow the tutorial from top to down. So the second link might better be removed.

To keep things at minimal, bold texts are probably not necessary. The quotation sort of already indicates it's not normal text. What do you think?

Comment (lorugant) [reply pages/vi/vi-github-and-markdown.md]: Made the changes...Please review. Thanks!!

**Comment by xyb994** [comment]
@lorugant Looks like you accidentally added [something extra](https://github.com/lorugant/lorugant.github.io/tree/lBranch/pages/techgenius/OLE) to this branch when you [commit last time](https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/1344/commits/6f068ddacf788b347833fe730160c1c606e641bc).

**Comment by lorugant** [comment]
I removed the second link in vi-github-and-markdown.md file and removed the bold lettering.I did not add anything.

**Comment by lorugant** [comment]
rawgit link: https://rawgit.com/lorugant/lorugant.github.io/lBranch/#pages/vi/vi-github-and-markdown.md

**Pull request closed without merging**